### ISSUE TITLE
NTR: Elevated uridine in urine

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4943]
**Preferred term label:**
Elevated urinary uridine

**Synonyms**
Elevated uridine in urine

**Definition (free text, please give PubMed ID)**
Increased levels of urinary uridine, a ribonucleoside composed of a molecule of uracil attached to a ribofuranose moiety via a beta‒N1-glycosidic bond.
Source: CHEBI:16704

**Parent term (use hpo.jax.org/app)**
HP_0031979 'Abnormal urine carbohydrate level'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618477 

See ticket: https://github.com/monarch-initiative/hpo-annotation-data/issues/387

**Comment by pnrobinson** [comment]
adding as HP_0032574

**Comment by pnrobinson** [comment]
done
